lesson2/hometask_TetyanaKarpova.rb

```diff
@@ -3,6 +3,8 @@ module M
   def self.included(base)
     base.include M::MInstanceMethods
     base.extend M::MClassMethods
+    puts base.singleton_methods              # => m_class_method
+    puts base.new.methods - Object.methods   # => m_instance_method
```

Comment:
use [#instance_methods](https://ruby-doc.org/core-2.2.0/Module.html#method-i-instance_methods) instead. There is no need to create instance.